docs/CONFIGURATION_SCHEMA.md

```diff
@@ -48,8 +48,9 @@ workerInstances:
   - ...
   
 # Services
-useMasterAsCompute: boolean [yes, no]               # Use master as compute instance, Default is no
-useMasterWithPublicIp: boolean [yes, no]            # Usage of public IP. Default is yes
+useMasterAsCompute: boolean [yes, "no"]               # Use master as compute instance, Default is no
+useMasterWithPublicIp: boolean ["yes", no]            # Usage of public IP. Default is yes
+useHostname: boolean [yes, "no"]
```

Comment:
must be **useHostnames** (instead of useHostname). A configuration option description is missing. Something like ```# Use hostnames instead ip adresses for ansible hosts file, needs a working instance DNS resolution (!)```